DOCS/license-inclusion-principles.md

```diff
@@ -1,24 +1,33 @@
-* Consider existing use of the license in wild (e.g., wide use of license or use of license in prominent projects)
-* If it is a newly drafted license, it is drafted such that it can be used by anyone (e.g., it is not company-specific or a "vanity" license)
-* Relevant input from the license steward (e.g., regarding name or if the license steward does not want the license added)
-* Relevant open source community factors
-
-Comments on license inclusion will be noted in the Github issue for the license (or reference to meeting minutes, as necessary). Some license requests may be decided solely via comments in the Github issue and some may involve discussion on the bi-weekly legal call. Thus it is recommended that requestors join the mailing list and calls to fully participate.
-
-
-## Backstory
+**Definitive factors:**
+* The submitted license must not match another license already on the SPDX License List as per the SPDX matching guidelines.
+* All OSI-approved licenses will be included on the SPDX License List.
+  * This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also att that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.
+* Binary licenses or licenses that do not provide for the avilabilty of the source code will not be included on the SPDX License List.
+  
+**Other factors (roughly in order of importance):**
+* The license substantially complies with one of the following open source definitions (even if not submitted for approval, as in the case of OSI) or these organization consider the license to meet their definition:
+  * [Open Source Definition (OSD)](https://opensource.org/osd) from the Open Source Initiative (OSI)
+  * [Free Software Definition](https://www.gnu.org/philosophy/free-sw.en.html) from the Free Software Foundation (FSF)
+  * [Open Source Hardware Definition](https://www.oshwa.org/definition/) from the Open Source Hardware Association (OSHWA)
+  * [Open Definition](http://opendefinition.org/od/2.1/en/) from the Open Knowledge Foundation (for open data)
+* The license is structured to be generally usabe by anyone. It is not a "vanity" license or specific to one project, consortium or corporation. 
+* The license has actual, substantial use such that it is likely to be encountered. Substantial use may be demonstreated via use in many projects, or in one or a few highly significant projects. 
+* The license has identifiable and stable text; it is not in the midst of drafting. The license steward is committed not modifying after addition to the list and to versioning new versions in the future.
+* If the license does not substantively comply with one of the above open source definitions, the license requires the availability of the source code with limited restrictions and meets other factors listed here.
```

Comment:
See earlier comments.    Data is not Source code, and we want to encourage sharing/automation here, not restrict.

Reply:
The Open Definition from the Open Knowledge Foundation addresses open content + open data, and that's one of the reference definitions listed above. So I think that should be fine?

Reply:
I agree with @kestewart - the SPDX list should be able to apply to data as well as software. The text here seems to be more restrictive and I don't think that was intentional.

Reply:
agree with @swinslow  - we have included various open * definitions as a first point. and if data is not source code, then this wouldn't apply to data...

Reply:
but we could also add "software" here, I suppose. the point is we are not opening the flood gate to any and all licenses. we still need to emphasize open / source available. I think that's been a clear intention from our discussions on calls and comments, no?

Reply:
actually, I don't think we need "software" here - isn't the open definitions enough to clarify that open data licenses are relevant? (not to mention, we already have some on the list)

Reply:
Some of the OSI-approved licenses are "vanity" licenses (that was the context where the term originally arose).

Reply:
"[T]he license requires the availability of the source code" - this would only be true of copyleft licenses; is that what is intended here?

Reply:
@richardfontana I think the intention is more to be clear that the license list is not meant for binary-only software licenses; and/or, that this is intended to pick up "source available but not FOSS" licenses. Perhaps something like the following instead?

* If the license does not substantively comply with one of the above open source definitions, then the license is primarily intended for free distribution of content (including, in the case of software, its source code) with limited restrictions, and also meets other factors listed here.

Reply:
@swinslow wording added